stanford-sw/test/src/edu/stanford/ItemInfoTests.java

```diff
@@ -141,6 +138,11 @@ public void testSkipBuildingFacet()
 	    // INDEX-168 Meyer no longer exists
 	    assertZeroResults(fldName, "\"MEYER\"");
 
+      // BIOLOGY, CHEMCHMENG, and MATH-CS are closed Libraries
+      assertZeroResults(fldName, "\"BIOLOGY\"");
+      assertZeroResults(fldName, "\"CHEMCHMENG\"");
+      assertZeroResults(fldName, "\"MATH-CS\"");
```

Comment:
alignment, and you can keep or remove the blank line before the close brace.  Not sure why it was in there before ...